src/HealthChecks.Solr/SolrOptions.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+namespace HealthChecks.Solr
+{
+    public class SolrOptions
+    {
+        public string Uri { get; private set; }
+        public string Core { get; private set; }
+        public int Timeout { get; private set; }

```

Comment:
Consider to use `TimeSpan`

Reply:
I use int because is the same type that have solrconnection. 
https://github.com/SolrNet/SolrNet/blob/master/SolrNet/Impl/SolrConnection.cs#L87

Reply:
I guessed. Your public api can always be improved by avoiding the ambiguous interpretation of `int` - seconds/milliseconds/nano? SolrNet documentation unfortunately leaves the reader alone with this question.